delphi-checks/src/main/resources/org/sonar/l10n/delphi/rules/community-delphi/FullyQualifiedImport.json

```diff
@@ -0,0 +1,18 @@
+{
+  "title": "Imports should be fully qualified",
+  "type": "CODE_SMELL",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "5min"
+  },
+  "code": {
+    "attribute": "CLEAR",
+    "impacts": {
+      "MAINTAINABILITY": "LOW"
+    }
+  },
+  "tags": [],
```

Comment:
Every rule should have at least one tag. We tend to use the same tags used in SonarSource's official plugins, which can be found [here](https://docs.sonarsource.com/sonarqube/latest/user-guide/rules/built-in-rule-tags/?gad_source=1). 

In this case, probably just `convention` is fine.

Reply:
Done!